modules/cudaoptflow/include/opencv2/cudaoptflow.hpp

```diff
@@ -444,7 +452,9 @@ class CV_EXPORTS_W NvidiaOpticalFlow_1_0 : public NvidiaHWOpticalFlow
         bool enableTemporalHints = false,
         bool enableExternalHints = false,
         bool enableCostBuffer = false,
-        int gpuId = 0);
+        int gpuId = 0,
+        Stream inputStream = Stream::Null(),
+        Stream outputStream = Stream::Null());
```

Comment:
According to other CUDA APIs this parameter is usually passed as `Stream&`.

Not sure about "const". /cc @asmorkalov

Reply:
> According to other CUDA APIs this parameter is usually passed as `Stream&`.
> 
> Not sure about "const". /cc @asmorkalov

When we do Stream x = Stream(), It creates a new stream and returns pointer to it right?

Reply:
See [here](https://github.com/opencv/opencv/blob/4.3.0/modules/core/include/opencv2/core/cuda.hpp#L1046).

Reply:
> See [here](https://github.com/opencv/opencv/blob/4.3.0/modules/core/include/opencv2/core/cuda.hpp#L1046).

Thanks. Modified it accordingly.